### PR TITLE
prep-javascript-objects - Farouk Ghodhbane

### PR DESCRIPTION
**Pull request by FaroukGhodhbane**
prep-javascript-objects

### CODE REVIEW

File: prep-javascript-objects/main.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+};
+console.log(person);
+
+const fullName = person.firstName + ' ' + person.lastName;
+console.log("The person's name is: ", fullName);
+
+person.job = 'Web Developer';
+console.log("The person's current job is: ", person.job);
+
+person.previousJob = 'Student';
```
Comment (BrettAlbright) [inline prep-javascript-objects/main.js]: this property should be added using bracket notation

File: prep-javascript-objects/main.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+console.log("The person's previous job is: ", person.previousJob);
```
Comment (BrettAlbright) [inline prep-javascript-objects/main.js]: This console log should be using bracket notation to access the previousJob property on the object

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
make sure to use bracket notation where indicated

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**